tests/unit_tests/ci/test_fisherz_test.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import pandas as pd
+
+from dodiscover.ci import FisherZCITest
+
+seed = 12345
+
+
+def test_fisher_z():
+    """Test Fisher Z test for Gaussian data."""
+    rng = np.random.RandomState(seed)
+    X = rng.randn(300, 1)
```

Comment:
Docstrings here that explain what is expected

Reply:
Done.